src/lib.rs

```diff
@@ -0,0 +1,181 @@
+pub const DEFAULT_CELL_AREA: f32 = 400.; // arbitrary value for now
+pub const DEFAULT_CELL_LEN: f32 = 20.; // arbitrary value for now
+
+#[derive(Clone, Copy, Debug)]
+pub struct GameCell {
+    start_x: f32,
+    start_y: f32,
+    alive: bool,
+}
```

Comment:
The internal representation for the game cell mixes UI stuff (x and y coordinates on screen), with the model for the game of life (which has fixed-size cells). This makes understanding the code more difficult, because you need to think about both domains at the same time.